signac/contrib/job.py

```diff
@@ -227,7 +231,11 @@ def _reset_sp(self, new_sp=None):
         self.reset_statepoint(new_sp)
 
     def update_statepoint(self, update, overwrite=False):
-        """Update the state point of this job.
+        """Change the state point of this job without affecting existing parameters.
+
+        Change the job id if necessary.
+        For more, see
+        `Modifying the State Point <https://docs.signac.io/en/latest/jobs.html#modifying-the-state-point>`_.
```

Comment:
This should use an intersphinx reference to `_project-job-statepoint-modify`.

See this guide: https://github.com/glotzerlab/signac-docs/wiki/How-to-cross-reference-links-between-package-docs-and-signac-docs-using-intersphinx